src/main/kotlin/io/libp2p/pubsub/gossip/GossipParams.kt

```diff
@@ -415,9 +415,22 @@ data class GossipPeerScoreParams(
  */
 class GossipTopicsScoreParams(
     private val defaultParams: GossipTopicScoreParams = GossipTopicScoreParams(),
-    private val topicParams: Map<Topic, GossipTopicScoreParams> = mapOf()
+    topicParamsMap: Map<Topic, GossipTopicScoreParams> = mapOf()
 ) {
+    val topicParams: MutableMap<Topic, GossipTopicScoreParams>
+    init {
+        this.topicParams = topicParamsMap.toMutableMap()
+    }
```

Comment:
nit: 
```suggestion
    val topicParams: MutableMap<Topic, GossipTopicScoreParams>= topicParamsMap.toMutableMap()
```

Reply:
hmm - I can't seem to get that to compile

Reply:
What's the error?

Reply:
Oh, weird - it was just a white space issue: 
<img width="819" alt="Screen Shot 2021-02-04 at 11 23 22 AM" src="https://user-images.githubusercontent.com/658601/106922879-7d3db880-66db-11eb-811f-054f17129afc.png">

Working now 😄